### PR TITLE
Fix slow test cases which depend on timeout seconds

### PR DESCRIPTION
**Pull request by supercaracal**
#780
#794

### CODE REVIEW

**Comment by byroot** [comment]
Master: `Finished in 1117.441769271 seconds.`
This branch: `Finished in 149.787086536 seconds.`

Nice 7-8x speedup here. 👏

File: test/lint/blocking_commands.rb [hunk 1]
```diff
@@ -1,5 +1,8 @@
 module Lint
   module BlockingCommands
+    TIMEOUT_SEC = 0.01
```
Comment (byroot) [inline test/lint/blocking_commands.rb]: I'm not a huge fan of `TIMEOUT_SEC`, as it's not very meaningful. What about `LOW_TIMEOUT` ?

File: test/lint/blocking_commands.rb [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+    DELAY_SEC = 0.05
```
Comment (byroot) [inline test/lint/blocking_commands.rb]: Similar issue with the naming here. I'd almost suggest not to constantize this and to use `LOW_TIMEOUT * 5` as an expression.

**Comment by supercaracal** [comment]
Hmm... Several errors occurred on CI but they didn't reproduce on my local environment. I'll try investigate them.

* https://travis-ci.org/redis/redis-rb/jobs/451166986
  * related Redis Sentinel
* https://travis-ci.org/redis/redis-rb/jobs/451167015
  * related #789

**Comment by supercaracal** [comment]
@byroot I have fixed them and CI has been success.

**Comment by byroot** [comment]
Many thanks for handling this!

**Comment by supercaracal** [comment]
https://github.com/redis/redis-rb/issues/774
fixed by 51ee07b

**Pull request merged**